templates/release-docs-changelog.yml

```diff
@@ -0,0 +1,59 @@
+spec:
+  inputs:
+    stage:
+      description: The pipeline stage where to perform the docs release
+      default: publish
+    runner:
+      description: GitLab CI runner tag
+      default: hetzner-amd-beefy
+    alpine-git-tag:
+      description: Tag for alpine/git container image
+      default: latest
+    changelog_file:
+      description: Remote changelog file
+      default: CHANGELOG.md
+---
+
+
+release:mender-docs-changelog:
+  stage: $[[ inputs.stage ]]
+  tags:
+    - $[[ inputs.runner ]]
+  needs: []
+  image:
+    name: alpine/git:$[[ inputs.alpine-git-tag ]]
+    entrypoint: [""]
+  rules:
+    # Only make available for stable branches
+    - if: '$CI_COMMIT_TAG =~ /^v?\d+\.\d+\.\d+$/'
```

Comment:
I missed this before: is this supposed to be branches or tags?

Reply:
I think branch, but it is just my guess